app/actions/VersionActions.js

```diff
@@ -50,10 +50,24 @@ function getWalletRPCVersionSuccess(getWalletRPCVersionResponse) {
     dispatch( { getWalletRPCVersionResponse: getWalletRPCVersionResponse, type: WALLETRPCVERSION_SUCCESS });
     const { address, port } = getState().grpc;
     const { requiredVersion } = getState().version;
-    if (requiredVersion != getWalletRPCVersionResponse.getVersionString()) {
-      var versionErr = 'Version not valid! got: ' + getWalletRPCVersionResponse.getVersionString() + ' expected: ' + requiredVersion +
-      '. Please change dcrwallet to expected version.';
-      dispatch( { error: versionErr, type: VERSION_NOT_VALID });
+    var versionErr = '';
+
+    if (!getWalletRPCVersionResponse.getVersionString()) {
+      versionErr = 'Unable to obtain wallet API version';
+    } else {
+      // discard the patch version before doing comparison
+      var grpcParts = getWalletRPCVersionResponse.getVersionString().split('.');
```

Comment:
you can just use the getWalletRPCVersionResponse.getMajor/Minor() instead of having to parse the string